### DISCUSSION TITLE
[Scam Alert] AI Marketing آی مارکتینگ

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#31] [Announcements]
<div dir="rtl">

**Project Name:** 

**نام پروژه:** 

**نماد پروژه:**

**اتفاقی که افتاده است:**
با توجه به پرداخت 5 درصدی پورسانت جهت معرفی و همچنین بازگردان مبلغ خرید تا 50 درصد این سایت نیز بهتر است در لیست هشدار قرار گیرد.

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار ریسک بسیار بالا

**نوع کلاهبرداری:** 
طرح پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
http://Ai.marketing
https://t.me/aimarketig_tat
https://t.me/aimarketig_tat/487
https://t.me/aimarketig_tat/490

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by saeedkousha** [comment]
با توجه به پرداخت 5 درصدی پورسانت جهت معرفی و همچنین بازگردان مبلغ خرید تا 50 درصد به نظر می رسد این سایت نیز بهتر است در لیست هشدار قرار گیرد.
